hugolib/site.go

```diff
@@ -976,7 +977,8 @@ func (s *Site) initializeSiteInfo() {
 		sectionPagesMenu:               lang.GetString("sectionPagesMenu"),
 		GoogleAnalytics:                lang.GetString("googleAnalytics"),
 		BuildDrafts:                    s.Cfg.GetBool("buildDrafts"),
-		canonifyURLs:                   s.Cfg.GetBool("canonifyURLs"),
+		CanonifyURLs:                   s.Cfg.GetBool("canonifyURLs"),
```

Comment:
I mean, instead of adding a set of path related fields to SIteInfo, I assume there is a PathSpec somewhere with these fields already.